### PR TITLE
Improve travis configuration

### PR DESCRIPTION
**Pull request by corneliusweig**
- upgrade to go 1.13
- set `GOPROXY` env var
- configure github releases on tags

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,29 +1,35 @@
...
+- osx
 
 go:
-  - "1.12.x"
+- 1.13.x
 
 env:
+  global:
   - GO111MODULE=on
+  - GOPROXY=https://proxy.golang.org
```
Comment (ferhatelmas) [inline .travis.yml]: passing by :wink: 

isn't it default? why is it needed?

Comment (corneliusweig) [reply .travis.yml]: Is it the default? I'm not sure.

Comment (ferhatelmas) [reply .travis.yml]: > The GOPROXY environment variable may now be set to a comma-separated list of proxy URLs or the special token direct, and its default value is now https://proxy.golang.org,direct

from release nodes of 1.13

Comment (ferhatelmas) [reply .travis.yml]: for convenience: https://golang.org/doc/go1.13#modules

Comment (corneliusweig) [reply .travis.yml]: Ah.. I also found something here https://arslan.io/2019/08/02/why-you-should-use-a-go-module-proxy/

Guess I will remove it then :)
Thanks for spotting this!

Comment (ferhatelmas) [reply .travis.yml]: happy to be helpful how minor it is.

**Pull request merged**